templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -751,18 +749,30 @@ contents:
     # Setup an exit trap to rollback on error
     handle_exit() {
       e=$?
-      [ $e -eq 0 ] && print_state && exit 0
+      tdir=$(mktemp -u -d -t "configure-ovs-$(date +%Y-%m-%d-%H-%M-%S)-XXXXXXXXXX")
```

Comment:
do you know is this picked up by sosreports?

Reply:
It's not. It would probably be a good idea to add it but from my experience we don't use this much if at all so it's there much a least resort.